scripts/batchaccession.py

```diff
@@ -262,10 +262,13 @@ def main(args_):
         to_accession = initial_check(args, accession_digits, oe_list, reference_number)
     else:
         to_accession = {}
+        print('\nIngested Info: Please check below if any info and characters are correct')
```

Comment:
Not to be too nit picky, but seeing this now, looks like you might have been going for “any info and characters are incorrect?’

Reply:
Looks like it - it should be for people to trace back when error happens.  What about
`Ingested info from CSVs: Please check the following if an error occurs` ?